src/index.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App';
+
+ReactDOM.render(<App />, document.getElementById("app"));
```

Comment:
Insert a final newline here and in the rest of the files. You can also enable this option in VSCode by adding `"files.insertFinalNewline": true` to your settings.json file